db/migrate/20210312140240_create_books.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def up
+    create_table :books do |t|
+      t.string :genre, null: false
+      t.string :author, null: false
+      t.string :image_url, null: false
+      t.string :title, null: false
+      t.string :publisher, null: false
+      t.string :year, null: false
+
+      t.timestamps
+    end
+  end
```

Comment:
Here you also have to define the down method. If you don't then the rollback of the migration won't work properly.